Lib/test/test_flufl.py

```diff
@@ -4,18 +4,29 @@
 class FLUFLTests(unittest.TestCase):
 
     def test_barry_as_bdfl(self):
-        code = "from __future__ import barry_as_FLUFL; 2 {0} 3"
+        code = "from __future__ import barry_as_FLUFL\n2 {0} 3"
```

Comment:
I am not familiar with `compile`. My question is when `flags` is passed, what's the meaning of the `import` here? Is it a must?

Reply:
It is redundant if flags is passed.

The bug is that this tests is failed if not pass flags.

Reply:
I know flags have to be passed. I just don't understand why not simply `'2 {0} 3'`?.

Reply:
The intention of the test is that `from __future__ import barry_as_FLUFL` should change grammar. But this doesn't work, and we need to pass flags as a temporary workaround. After fixing this bug we could remove flags.

Reply:
gotcha. I wrongly thought the `from __future__ import ...` doesn't work in `compile` and `flags` must be passed.